README.md

```diff
@@ -1,5 +1,7 @@
 # Status Ideas
 
+Read about the [process around Idea Proposals](https://docs.google.com/document/d/1CaFM2ZXGOKf05_LXMPJeNNy5qJOdAq91EF2Gn2QUBFI/edit#heading=h.cb5r1eeit5n) in the "Status Organisational Design" document.

```

Comment:
Link is broken for me:

> Sorry, the file you have requested has been deleted.

Reply:
Maybe link to this one? https://wiki.status.im/Status_Organisational_Design